cmd/cluster/dynatrace/logsCmd.go

```diff
@@ -14,44 +15,75 @@ var (
 	since         int
 	contains      string
 	sortOrder     string
+	clusterID     string
 	namespaceList []string
 	nodeList      []string
 	podList       []string
 	containerList []string
 	statusList    []string
 )
 
+const (
+	logsCmdDescription = `
+  Fetch logs of current cluster context (by default) from Dynatrace and display the logs like oc logs.
+
+  This command also prints the Dynatrace URL and the corresponding DQL in the output.
+
+`
+
+	logsCmdExample = `
+  # Get the logs of HCP cluster in current cluster context.
+  $ osdctl cluster dynatrace logs
+
+  # Get the logs of a specific HCP cluster
+  $ osdctl cluster dynatrace logs -c <cluster-id>
```

Comment:
I think the `-c` flag is used for container. This can cause confusion for the usage where -c is valid on different CLI but can refer to either container (oc) or cluster(rosa/ocm)
I would recommend not to include `-c` at all for any flags to avoid any confusion. While testing, I thought the `-c` flag to be cluster ID initially 😓